app/models/feeds/megaphone_feed.rb

```diff
@@ -41,6 +41,10 @@ def set_defaults
     super
   end
 
+  def serve_drafts
+    publish_integration?
+  end
```

Comment:
megaphone feeds will allow draft audio, but only if it is a valid, saved, enabled feed